recipes/skaware/skalibs.inc

```diff
@@ -0,0 +1,66 @@
+DESCRIPTION = "skalibs is a package centralizing the free software / open source C development files used for building all software at skarnet.org"
+LICENSE = "ISC"
+
+SRC_URI = "http://skarnet.org/software/${PN}/${PN}-${PV}.tar.gz"
+
+inherit c make
+
+endianness = "${@['big', 'little'][d.get('HOST_ENDIAN') == 'l']}"
+
+SRC_URI += "file://sysdeps file://sysdeps.h"
+do_configure[prefuncs] += "do_configure_sysdeps"
+do_configure_sysdeps() {
+	rm -rf ${B}/sysdeps
+	mkdir -p ${B}/sysdeps
+	echo "" > ${B}/sysdeps/rt.lib
+	echo "" > ${B}/sysdeps/socket.lib
+	echo "" > ${B}/sysdeps/sysclock.lib
+	echo "" > ${B}/sysdeps/tainnow.lib
+	echo "" > ${B}/sysdeps/util.lib
+	echo "${HOST_ARCH}" > ${B}/sysdeps/target
+	cp ${SRCDIR}/sysdeps ${SRCDIR}/sysdeps.h ${B}/sysdeps/
+
+	sed -e 's/^\(target\):. */\1: ${HOST_ARCH}/' \
+	    -e 's/^\(endianness\): .*/\1: ${endianness}/' \
+	    -i ${B}/sysdeps/sysdeps
+	sed -e 's/^\(#define SKALIBS_TARGET\) .*/\1 "${HOST_ARCH}"/' \
+	    -e 's/^\(#define SKALIBS_ENDIANNESS\) .*/\1 "${endianness}"/' \
+	    -i ${B}/sysdeps/sysdeps.h
+}
+
+RECIPE_FLAGS = "skalibs_ipv6"
+do_configure[prefuncs] += "${do_configure_ipv6}"
+do_configure_ipv6 = ""
+do_configure_ipv6:USE_skalibs_ipv6 = "do_configure_ipv6_enable"
+do_configure_ipv6_enable() {
+	sed -e 's/^\(ipv6\):. */\1: yes/' \
+	    -i ${B}/sysdeps/sysdeps
+	sed -e 's/^#undef \(SKALIBS_HASIPV6\)/#define \1/' \
+	    -i ${B}/sysdeps/sysdeps.h
+}
+
+do_configure() {
+	./configure \
+		--prefix=${prefix} \
+		--dynlibdir=${sharedlibdir} \
+		--libdir=${libdir} \
+		--includedir=${includedir} \
+		--datadir=${datadir} \
+		--sysdepdir=${libdir}/skalibs/sysdeps \
+		--with-sysdeps=${B}/sysdeps \
+		--disable-ipv6 \
```

Comment:
Good question.  The SKALIBS_HASIPV6 is bogus.  IPv6 support is controlled with ./configure arguments. I will fix it.